send-emails-from-a-service/main.bal

```diff
@@ -0,0 +1,173 @@
+import ballerina/email;
+import ballerina/http;
+import ballerina/log;
+import ballerina/uuid;
+
+type Patient record {|
+    string name;
+    string dob;
+    string ssn;
+    string address;
+    string phone;
+    string email;
+|};
+
+type Doctor record {|
+    string name;
+    string hospital;
+    string category;
+    string availability;
+    decimal fee;
+|};
+
+type Appointment record {|
+    int appointmentNumber;
+    Doctor doctor;
+    Patient patient;
+    boolean confirmed;
+    string hospital;
+    string appointmentDate;
+|};
+
+type PaymentSettlement record {|
+    int appointmentNumber;
+    Doctor doctor;
+    Patient patient;
+    decimal fee;
+    boolean confirmed;
+    string card_number;
+|};
+
+type Payment record {|
+    int appointmentNo;
+    string doctorName;
+    string patient;
+    decimal actualFee;
+    int discount;
+    decimal discounted;
+    string paymentID = uuid:createType4AsString();
+    string status;
+|};
+
+type ReservationRequest record {|
+    record {|
+        *Patient;
+        string cardNo;
+    |} patient;
+    string doctor;
+    string hospital_id;
+    string hospital;
+    string appointment_date;
+|};
+
+type ReservationResponse record {|
+    int appointmentNo;
+    string doctorName;
+    string patient;
+    decimal actualFee;
+    int discount;
+    decimal discounted;
+    string paymentID;
+    string status;
+|};
+
+configurable int port = 8290;
+configurable string hospitalServicesBackend = "http://localhost:9090";
+final http:Client hospitalServicesEP = check initializeHttpClient(hospitalServicesBackend);
+
+configurable string host = "smtp.gmail.com";
+configurable string senderAddress = "rominxd97@gmail.com";
+configurable string appPassword = "ztqmiijrbiiosrrw";
```

Comment:
group the configurables and other variable declarations to improve the readability